geomstats/test_cases/geometry/fiber_bundle.py

```diff
@@ -477,15 +482,19 @@ def tangent_riemannian_submersion(self, tangent_vec, base_point):
         return 2 * Matrices.to_symmetric(product)
 
     def horizontal_lift(self, tangent_vec, base_point=None, fiber_point=None):
+        if base_point is None and fiber_point is None:
+            raise ValueError(
+                "Either a point (of the total space) or a "
+                "base point (of the base manifold) must be "
+                "given."
+            )
+
         if base_point is None:
-            if fiber_point is not None:
-                base_point = self.riemannian_submersion(fiber_point)
-            else:
-                raise ValueError(
-                    "Either a point (of the total space) or a "
-                    "base point (of the base manifold) must be "
-                    "given."
-                )
+            base_point = self.riemannian_submersion(fiber_point)
```

Comment:
NIT: We could use total_point instead of fiber_point, so that the points mirror the names of the spaces:
- total_point belongs to total_space,
- base_point belongs to base_space,
and "fiber_point" can confuse people as it seems that it refers to a point on the fiber (i.e., to a group element), as opposed to a point in the fiber bundle (which is an element of the total space)